cli/src/utils/compareSchema.ts

```diff
@@ -51,45 +59,6 @@ export function compareSchemas(
     for (const column of deletedColumns) {
       edits.push({ drop_column: { table, column } });
     }
-
-    // Compare column properties
-    for (const column of targetColumns) {
```

Comment:
In order to get this working properly we would need to:;
1. add the correct up/down fields for the migrations
2. split apart add columns and alter columns into separate migrations to run because of the limitation of altering new objects from pgroll

I removed it for now